src/tools/rbd/action/Status.cc

```diff
@@ -24,7 +24,7 @@ namespace {
 const std::string IMAGE_CACHE_STATE = ".librbd/image_cache_state";
 
 struct ImageCacheState {
-  bool present;
+  bool present = false;
```

Comment:
@dillaman  I did a test, it won't be initialized to false by default.

Reply:
Might as well default initialize `clean` and `size` as well.